popularity/lib/popularity.dart

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+library package_popularity;
+
+import 'package:json_annotation/json_annotation.dart';
+
+part 'popularity.g.dart';
+
+@JsonSerializable()
+class PackagePopularity extends Object with _$PackagePopularitySerializerMixin {
+  @JsonKey(name: 'date_first')
+  final DateTime dateFirst;
```

Comment:
Wouldn't be `firstDate` and `lastDate` or maybe `from` and `to` be better?

Reply:
Eh...I like 'em how they are 🤷‍♂️